src/components/modules/auth/ui/pages/ResetPasswordClient.tsx

```
@@ -4,31 +4,26 @@ import { useSearchParams, useRouter } from "next/navigation";
 import { ResetPasswordForm } from "../forms/ResetPasswordForm";
 import { supabase } from "@/lib/supabase";
-import {
-  CardHeader,
-  CardTitle,
-  Card,
-  CardContent,
-} from "@/components/ui/card";
+import { CardHeader, CardTitle, Card, CardContent } from "@/components/ui/card";
 
 export function ResetPasswordClient() {
   const searchParams = useSearchParams();
```

ESLint

[error] 'searchParams' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)